.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: goreleaser
+
+env:
+  GO_VERSION: '1.22'
+  GO111MODULE: on
+
+on:
+  push:
+    tags:
+      - "v*"
```

Comment:
Does this potentially support full regex (`v\d+\.\d+\.\d+`) or even `v*.*.* just to reduce the chance of a miss release.